tests/e2e/conftest.py

```diff
@@ -436,7 +442,7 @@ def run_cli(self, arguments: List[str], *, verbosity: int = 0) -> SysCap:
                 sleep(delay)
                 delay *= 2
                 continue
-            else:
+            elif wait_for_exit_code:
```

Comment:
sometimes, when we execute an interactive command, we don't want to wait until it returns (it won't)